contrib/drivers/ceph/ceph.go

```diff
@@ -382,25 +408,25 @@ func (d *Driver) visitSnapshot(snapID string, fn func(imgName string, img *rbd.I
 			log.Error("When GetSnapshotNames:", err)
 			continue
 		}
-
-		snapName := opensdsPrefix + snapID
 		for _, snapInfo := range snapInfos {
-			if snapName == snapInfo.Name {
-				return fn(imgName, img, &snapInfo)
+			name := ParseName(snapInfo.Name)
+			if snapID == name.GetUUID() {
+				return fn(in, img, &snapInfo)
 			}
 		}
 	}
-	return fmt.Errorf("Not found the snapshot(%s)", snapID)
+	reason := fmt.Sprintf("Not found the snapshot(%s)", snapID)
+	return errors.New(reason)
 }
 
 func (d *Driver) PullSnapshot(snapID string) (*model.VolumeSnapshotSpec, error) {
 	return nil, fmt.Errorf("Ceph PullSnapshot has not implemented yet.")
 	/*
-		if err := d.init(); err != nil {
+		if err := d.initConn(); err != nil {
```

Comment:
Again why is this block of code commented out and the error message stated that this method is not implemented?

Reply:
@xing-yang This commented out interface doesn't  used in current version. and this interface will be mofidied in the future,  so we just commented out it here.

Reply:
It looks like @xxwjj replied here (I got an email notification) but can't find it here:).  If it is not used now but will be used in the future, can you please add a "NOTE" here to explain?  Usually unused code should be removed.  If you want to keep it, please add a NOTE explaining why.